web-components/src/components/table/ActionsTable.tsx

```diff
@@ -164,14 +173,24 @@ const ActionsTable: React.FC<{
                   rowsPerPageOptions={[5, 10]}
                   rowsPerPage={rowsPerPage}
                   onChangeRowsPerPage={e => {
-                    setRowsPerPage(parseInt(e.target.value, 10));
+                    const newRowsPerPage = parseInt(e.target.value, 10);
+                    setRowsPerPage(newRowsPerPage);
                     setOffset(0);
+                    onTableChange &&
```

Comment:
We're now calling a callback with pagination params when rowsPerPage is updated.

Reply:
extract this logic to a handler function?